plugins/techdocs/src/reader/transformers/styles/rules/typeset.ts

```diff
@@ -38,8 +38,8 @@ ${headings.reduce<string>((style, heading) => {
   const calculate = (value: typeof fontSize) => {
     let factor: number | string = 1;
     if (typeof value === 'number') {
-      // 60% of the size defined because it is too big
-      factor = (value / 16) * 0.6;
+      // convert px to rem
+      factor = value / 16;
```

Comment:
Based on @freben question above, did you plan to use the default value (`DEFAULT_HTML_FONT_SIZE`) here @camilaibs? if so I guess this need to be updated to something like: 

```suggestion
      factor = value / theme.typography.htmlFontSize;
```

Reply:
Done, thanks!